package.json

```diff
@@ -40,7 +40,8 @@
     "mustache": "^2.2.1",
     "mutationobserver-shim": "^0.3.1",
     "pbf": "^1.3.2",
-    "point-in-polygon": "1.0.0",
+    "point-geometry": "^0.0.0",
+    "point-in-polygon": "steve9164/point-in-polygon",
```

Comment:
Maybe a fork is the wrong thing, but before I was doing point-in-polygon calculations with a ring and a point from GeoJSON data. The ring was an array of `[x,y]` coordinates. Now I'm doing the calculations with a ring that isn array of `Point(x,y)` coordinates. Basically, I changed every `ring[i][0]` to `ring[i].x` and the same for `[1] -> .y` so it would work with the new data layout.

The package is just 1 function. I could instead copy and paste it into `MapboxVectorTileImageryProvider.js` and comment that it is from `point-in-polygon` as we can still include the licence for that.